src/app/endround/endround.component.ts

```diff
@@ -0,0 +1,19 @@
+import { Component, Input, SimpleChanges } from '@angular/core';
+
+@Component({
+  selector: 'app-endround',
+  templateUrl: './endround.component.html',
+  styleUrls: ['./endround.component.scss'],
+})
+
+export class EndroundComponent {
+  @Input() match!: any;
+  teamWon = 0;
+  ngOnChanges(changes: SimpleChanges) {
+      if (changes["match"]) {
+        const match = changes["match"].currentValue;
+        if(match.attackersWon == true && match.teams[0].isAttacking == true) { this.teamWon = 0; }
```

Comment:
This logic is faulty. With this, team 0 can only win if attackers win and they are attackers.
Team 1 wins in all other cases.

Reply:
This is the current tree:

Attackers win:
  Is team 0 attack? 0 wins
  Is team 0 defense? 1 wins
Defenders win:
Team 1 wins

Reply:
The tree needs to look like this:

Attackers win:
  Is team 0 attack? 0 wins
  Is team 0 defense? 1 wins
Defenders win:
  Is team 0 attack? 1 wins
  Is team 0 defense? 0 wins

Reply:
resolved in 23bb25d